### PR TITLE
ET-587 - add check for classic editor option to load assets with tick…

### PR DESCRIPTION
**Pull request by jesseeproductions**
…et block

_Ref:_ [C#ET-587](https://moderntribe.atlassian.net/browse/ET-587)

### CODE REVIEW

File: src/Tribe/Tickets_View.php [hunk 1]
```diff
@@ -979,6 +979,12 @@ public function get_tickets_block( $post ) {
 		/** @var \Tribe__Tickets__Editor__Blocks__Tickets $blocks_tickets */
 		$blocks_tickets = tribe( 'tickets.editor.blocks.tickets' );
 
+		// load assets if classic editor is active
+		$option = get_option( 'classic-editor-replace' );
```
Comment (jesseeproductions) [inline src/Tribe/Tickets_View.php]: - I am not sure if this check is required.
- We could always load the assets here as there are other plugins to disable gutenberg besides the classic editor. 
- Not sure if there is a scenario where we load assets twice and it creates a performance issue.

File: readme.txt [hunk 2]
```diff
@@ -117,6 +117,10 @@ Currently, the following add-ons are available for Event Tickets:
 
 == Changelog ==
 
+= [4.11.3] TBD =
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
= [4.11.2] TBD =
```

File: src/Tribe/Tickets_View.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/Tickets_View.php]: It sounds like we should always load the assets (once) here. Doesn't the assets loader only ever load it once?

Comment (jesseeproductions) [reply src/Tribe/Tickets_View.php]: I believe so. I updated the PR to always load them. Ready for another review.

**Pull request merged**